libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -41,6 +41,11 @@
 // Device id
 static NSString* uid = nil;
 
+static NSMutableSet *features;
```

Comment:
This variable is only ever used in the singleton context.  It should be represented as a property on the singleton.